vaadin-iron-list-flow-parent/vaadin-iron-list-flow/src/main/java/com/vaadin/flow/component/ironlist/IronList.java

```diff
@@ -72,13 +72,16 @@
  * @see <a href=
  *      "https://www.webcomponents.org/element/PolymerElements/iron-list">iron-list
  *      webcomponent documentation</a>
+ *
+ * @deprecated {@code IronList} is deprecated in favor of {@code VirtualList}
```

Comment:
```suggestion
 * @deprecated since Vaadin 21, {@code IronList} is deprecated in favor of {@code VirtualList}
```

Reply:
Is it possilbe to turn `VirtualList` into a clickable link in JavaDoc? Something like `{@link com.vaadin.flow.component.virtuallist.VirtualList}`?

Reply:
Can't seem to get a link like that working properly